Item(by='djm_', descendants=None, kids=None, score=None, time=1609669847, title=None, item_type='comment', url=None, parent=25620866, text='Yes, running pytest with the --pdb flag will drop you into the debugger on an unhandled exception which gets a comparable workflow but it&#x27;s not quite the same as, a) writing tests to file first is not repl-driven development, and b) you generally have to think about doing it first.<p>In an ideal repl-driven world you could write the test in the repl entirely and commit it to disk once you&#x27;re ready.')